### PR TITLE
NoMerge: For review, apiman offline blog

### PR DESCRIPTION
**Pull request by msavy**
The blog's CSS needs fixing first as [code callouts](http://asciidoctor.org/docs/user-manual/#copy-and-paste-friendly-callouts) aren't showing at the moment. See also http://asciidoctor.org/docs/user-manual/#callout-icons

@apiman/developers - Any reviews welcome :)

See rendered: https://github.com/msavy/apiman.github.io/blob/locked-down-network/_blog-src/_posts/2016-04-04-locked-down-network.adoc

### CODE REVIEW

File: _blog-src/_posts/2016-04-04-locked-down-network.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+```
+<1> Use an internal https://maven.apache.org/repository-management.html#Available_Repository_Managers[repository or mirror] to deploy your own (and community) plugins to. This is likely the best option for many enterprises who may already have such infrastructure. Join multiple repositories with commas.
+<2> Mount a repository to a shared volume or filesystem on every machine running apiman.
+<3> Copy a valid Maven formatted repository containing your plugins to `~/.m2`, where apiman will look by default.
+
+TIP: *Advanced:* You can replace the https://github.com/apiman/apiman/blob/master/gateway/engine/core/src/main/java/io/apiman/gateway/engine/impl/DefaultPluginRegistry.java[default registry] entirely with an implementation of your own and set it via `apiman-gateway.plugin-registry` in `apiman.properties`.
+
+=== Registry (Plugin Catalogue)
+
+Our community releases contain a plugin registry which downloads a curated list of plugins. This lets the API Manager you have a nice prepopulated list of plugins to install, rather than having to type the coordinates manually. To preserve this functionality you can simply download the registry file and make it available internally:
```
Comment (EricWittmann) [inline _blog-src/_posts/2016-04-04-locked-down-network.adoc]: "This lets the API Manager you have a" ... drop the 'you'?

File: _blog-src/_posts/2016-04-04-locked-down-network.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+```properties
+apiman-manager.plugins.registries=http://rawgit.com/apiman/apiman-plugin-registry/master/registry.json <1>
+
+apiman-manager.plugins.registries=file:///path/to/my/registry.json <2>
```
Comment (EricWittmann) [inline _blog-src/_posts/2016-04-04-locked-down-network.adoc]: TIP: when running in wildfly/eap you can use appropriate platform system properties to make the config more portable:

```
apiman-manager.plugins.registries=file:///${jboss.server.config.dir}/apiman-plugin-registry.json
```

**Comment by msavy** [comment]
image

**Comment by kahboom** [comment]
@msavy - Thanks, taking a look.

**Comment by msavy** [comment]
@kahboom Do you want to merge your CSS first or shall I just glue it in here?

**Comment by msavy** [comment]
Added CSS fixes for conum stuff

**Comment by msavy** [comment]
image

The border around the captions looks weird IMO, but we can fix it later.

The conums themselves look great!

**Comment by EricWittmann** [comment]
Ready to merge now?

**Comment by msavy** [comment]
Yes!

**Comment by msavy** [comment]
Well, I'll regen first

**Pull request merged**